src/main/java/ch/njol/skript/SkriptCommand.java

```diff
@@ -54,8 +55,10 @@
 import java.util.Set;
 import java.util.stream.Collectors;
 
+import static ch.njol.skript.localization.PluralizingArgsMessage.format;
```

Comment:
don't import this statically
it was fine before, and now I think it looks like `format` is a method on this class instead.